prisma/migrations/20240216094338_add_user_model/migration.sql

```diff
@@ -0,0 +1,22 @@
+-- CreateEnum
+CREATE TYPE "Role" AS ENUM ('Admin', 'SuperAdmin', 'Customer', 'Trainer');
+
+-- CreateEnum
+CREATE TYPE "UserStatusEnum" AS ENUM ('Active', 'Pending');
```

Comment:
Should we only have two states?
What should be the state after user deactivation?

Reply:
As i remember we discuss  this case and  user status changes from "pending" to "active" after "superAdmin" approves the user. "deactivated" status should have a users who have a monthly or yearly membership package.

Reply:
Արթուր ջան, ես լավ չեմ հիշում, բայց ցանկացած տեսակի user պիտի կարողանանք ջնջել (բացի superAdmin֊ից), ուրեմն պիտի status լինի դրա համար։

Reply:
I suggest to have a call tomorrow and discuss the details.